### ISSUE TITLE
Configuration option not longer needed

### ISSUE THREAD

**Issue by smartinez290** [haffonist/igamt#2773]
There is no longer a need to have the option of including "Duplicate OBX datatype value when no flavor has been selected" in the configuration export for the co-constraints table. The tool no longer allows the user to leave the Data type flavor column empty. 

![image](https://user-images.githubusercontent.com/14842422/35700849-2b774d30-0762-11e8-8467-936a29467645.png)

**Comment by maxence-lefort** [comment]
@smartinez290 @haffonist What about existing IG?

**Comment by smartinez290** [comment]
Good point! maybe we still have to give them the option of not including the duplicates. Also, The tool even requires a flavor for primitive DTs.... Maybe this will change with the new co-constraint implementation.

**Comment by haffo** [comment]
@Jungyubw I beleive you add a boostrap for existing missing obx-2 ?

**Comment by afrold** [comment]
@Jungyubw  @maxence-lefort: by default we want to select the OBX-2 value in the OBX-2 Flavor.  
Please write the boostrap for exisiting co-constraints.

**Comment by afrold** [comment]
@maxence-lefort  Here is the text we want for now:
Label: Grey out OBX-2 Flavor column
Hover: When OBX-2 Value and OBX-2 Flavor are the same, check this box to grey out the OBX-2 Flavor  column.

**Comment by smartinez290** [comment]
@maxence-lefort it looks like this was done only on the compact configuration and not in the verbose configuration.

![image](https://user-images.githubusercontent.com/14842422/37837369-69859208-2e8b-11e8-9b57-2f896b739394.png)
